pema_docker_image/modules/initialize.bds

```diff
@@ -24,7 +24,7 @@ string{} readParameterFile(string parameter_file) {
       } else {
          string pname, pvalue
          
-         (pname, pvalue) = line.split('\t')       
+         (pname, pvalue) = line.split('=')
```

Comment:
:muscle:

Reply:
@savvas-paragkamian just for the record, did you run any sannity check using the `=` case? 
I cannot think of any reason of having an issue but just in case.

Reply:
No I didn't run a test. Maybe I should set up Docker to test it.

Reply:
No worries! 
I can merge your PR and run the sanity checks locally.